microdnf/commands/reinstall/reinstall.cpp

```diff
@@ -89,7 +89,12 @@ void CmdReinstall::run(Context & ctx) {
         result_pset |= solv_query.get_package_set();
     }
 
-    download_packages(result_pset, nullptr);
+    std::vector<libdnf::rpm::Package> download_pkgs;
+    download_pkgs.reserve(result_pset.size());
+    for (auto package : result_pset) {
+        download_pkgs.push_back(std::move(package));
+    }
+    download_packages(download_pkgs, nullptr);
```

Comment:
Why you use "." for one command and nullptr for another one?

Reply:
The `nullptr` means the package cache. The `download` command differs. It is used to download package to the current working directory.